src/codeFly/fileSystem/Repository.java

```diff
@@ -84,9 +84,11 @@ public String getQuestionDescription(int qNum) throws IOException {
 
         List<String> file = Files.readAllLines(Paths.get(path));
         StringBuilder description = new StringBuilder();
-        for (int i = 1; i < file.size(); i++) {
-            description.append(file.get(i));
+        for (int i = 1; i < file.size() - 1; i++) {
+            description.append(file.get(i)).append("\n");
```

Comment:
No need to add a new line character in the end. It is handled in the front-end code.